packages/angular-standalone-test-app/src/preview-examples/toast-custom.ts

```diff
@@ -17,24 +17,47 @@ import { ToastService } from '@siemens/ix-angular';
   selector: 'app-example',
   imports: [IxButton],
   template: `
-    <ix-button (click)="showToastMessage()">Trigger toast</ix-button>
-    <ng-template #customToast let-toast>
+    <ix-button (click)="showToastMessage()" style="margin-right: 0.5rem">
+      Trigger toast with custom message
+    </ix-button>
+    
+    <ix-button (click)="showToastMessageAction()">
+      Trigger toast with action button
+    </ix-button>
+
+    <ng-template #customToastMessage let-toast>
+      <div>This message is from template</div>
+    </ng-template>
+
+    <ng-template #customToastAction let-toast>
       <div>
-        <div>Custom toast message</div>
-        <ix-button (click)="toast.close('Action')">Action</ix-button>
+        <ix-button ghost icon="undo" (click)="toast.close('Action')">Undo</ix-button>
       </div>
     </ng-template>
   `,
 })
 export default class ToastCustom {
-  @ViewChild('customToast', { read: TemplateRef })
+  @ViewChild('customToastMessage', { read: TemplateRef })
   customToastRef!: TemplateRef<any>;
 
+  @ViewChild('customToastAction', { read: TemplateRef })
+  customToastActionRef!: TemplateRef<any>;
+
   constructor(private readonly toastService: ToastService) {}
 
   async showToastMessage() {
     this.toastService.show({
+      title: 'Toast headline',
+      message: 'Toast message text',
+      type: 'success',
+    });
+  }
+
+  async showToastMessageAction() {
+    this.toastService.show({
+      title: 'Toast headline',
       message: this.customToastRef,
```

Comment:
Please add the custom message to the first toast for all examlpes
```suggestion
      title: 'Toast headline',
      message: this.customToastRef,
      type: 'success',
    });
  }

  async showToastMessageAction() {
    this.toastService.show({
      title: 'Toast headline',
```

Reply:
Updated all preview examples